utils/test/jest.setup.js

```diff
@@ -9,3 +9,8 @@
 import 'jest-styled-components';
 
 import '@testing-library/jest-dom';
+
+/* eslint-disable n/prefer-global/text-encoder */
+import { TextEncoder } from 'node:util';
+
+global.TextEncoder = TextEncoder;
```

Comment:
Needed due to the addition of `renderToString` in `StyledTextInput`.

Reply:
Looks like this matches browser behavior. 😰 I always get nervous about using node utils in tests when the component will leverage browser APIs in practice.